### DISCUSSION TITLE
Disable selection on click

### DISCUSSION THREAD

**Discussion by madalinul** [ThibaultJanBeyer/DragSelect#173] [General]
I have a gallery with lightbox functionality and I use drag to select the images in the gallery. My main issue is that for the user to open the lightbox it needs to click the image and that selects the image.
Is there a way to only select elements with dragging and not clicking?

**Comment by ThibaultJanBeyer** [comment]
Not the cleanest, but this would work:

```JavaScript
ds.subscribe("predragstart", ({ event }) => {
  if(ds.getSelectables().includes(event?.target) && !event?.target?.classList.contains("ds-selected")) {
    ds.stop();
    setTimeout(() => ds.start());
  }
})
```
